index.bs

```diff
@@ -39,28 +39,22 @@ Markup Shorthands: css off, markdown on
 
 <pre class="anchors">
 
-<!-- spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/ -->
-spec: HTML51; urlPrefix: http://www.w3.org/TR/html51/; for: web
+spec: HTML; urlPrefix: https://html.spec.whatwg.org/multipage/
```

Comment:
I could also switch this to HTML52 and https://w3c.github.io/html/. IMO, it's not quite as good as the WHATWG's version, but it fixes many mistakes in HTML5.1, which will save folks time reading your crosslinks.